src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(SolutionDir)/Config/CS.props" />
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project=".\obj\WorkspaceDependencyViewExtension.csproj.nuget.g.props" Condition="Exists('.\obj\WorkspaceDependencyViewExtension.csproj.nuget.g.props')" />
```

Comment:
@alfredo-pozo I missed this - can you find any docs about this?

Reply:
Yes, check https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty 
![image](https://user-images.githubusercontent.com/47230272/96660313-d4777300-130e-11eb-9514-af5b51c9358b.png)